astrophot/models/psf_model_object.py

```diff
@@ -0,0 +1,31 @@
+import torch
+
+from .model_object import Component_Model
+from ..utils.decorators import default_internal
+
+
+__all__ = ["PSF_Model"]
+
+
+class PSF_Model(Component_Model):
+    """Prototype point source (typically a star) model, to be subclassed
+    by other point source models which define specific
+    behavior. Mostly this just requires that no standard PSF
```

Comment:
"Mostly this just requires that no standard PSF convolution is applied to this model as that is to be handled internally by the model."

This sentence is a little awkward, and may leave a prospective user confused about who/what is the active agent that would apply a convolution to the model.

Reply:
I see what you mean, this doc seems more for myself than a prospective user... I'll update it to make more sense for people other than myself